.github/workflows/regression-tests.yml

```diff
@@ -200,6 +200,9 @@ jobs:
       continue-on-error: ${{matrix.may_fail || false}}
       timeout-minutes: 15
       run: |
+        if [ "${{matrix.self-hosted}}" == "true" ]; then
+          module load "${{ matrix.sim-version }}"
```

Comment:
Are the Modulefiles themselves somewhere we could access?

Reply:
No. (But then, they are not fancy either. Effectively just a PATH modification and potentially a small set of further environment variables needed for the tool.)